tests/waku_rln_relay/test_rln_group_manager_onchain.nim

```diff
@@ -396,17 +433,26 @@ suite "Onchain group manager":
     let fut = newFuture[void]()
 
     proc callback(registrations: seq[Membership]): Future[void] {.async.} =
+      
+      when defined(rln_v2):
+        require:
+          registrations[0].rateCommitment == RateCommitment(idCommitment: idCommitment, userMessageLimit: UserMessageLimit(1))
```

Comment:
Maybe interesting to add a simple validation to avoid possible crash
```suggestion
          registrations.len == 1
          registrations[0].rateCommitment == RateCommitment(idCommitment: idCommitment, userMessageLimit: UserMessageLimit(1))
```